analytics-provider/src/main/java/org/digma/intellij/plugin/analytics/AnalyticsProvider.java

```diff
@@ -28,11 +28,12 @@
 
 public interface AnalyticsProvider extends Closeable {
 
+    String getApiUrl();
```

Comment:
not sure that this should be an API method because its used only with the concrete class RestAnalyticsProvider so it can be a method only on RestAnalyticsProvider.
its a source for errors because the source of truth for api url is the settings